lib/feature_flagger/model.rb

```diff
@@ -32,25 +36,37 @@ def feature_flagger_identifier
       public_send(self.class.feature_flagger_model_settings.identifier_field)
     end
 
+    def feature_flagger_name
+      self.class.feature_flagger_model_settings.entity_name
+    end
+
+    def feature_flagger_key
+      "#{feature_flagger_name}:#{feature_flagger_identifier}"
+    end
+
     module ClassMethods
       def feature_flagger
         raise ArgumentError unless block_given?
         yield feature_flagger_model_settings
       end
 
-      def released_id?(resource_id, *feature_key)
+      def released_id?(resource_id, resource_key, *feature_key)
```

Comment:
I think we don't need `resource_key` as we know the resource and we know the resource_id